publish.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
 PUBLISH_MODE=1 yarn build
-cp .htaccess build/.
-rsync -av --delete-after build/ crozet@ssh.cluster003.hosting.ovh.net:/home/crozet/wgmath/
+# cp .htaccess build/.
+# rsync -av --delete-after build/ crozet@ssh.cluster003.hosting.ovh.net:/home/crozet/wgmath/
+
+sudo cp .htaccess build/.
+sudo cp -r build/* /var/www/html/test-wgmath/
+sudo cp .htaccess /var/www/html/test-wgmath/
+
+sudo systemctl restart apache2
```

Comment:
These changes should be reverted. Though we could add a new `start_local_server.sh` script with that content.